request_options.go

```diff
@@ -56,7 +56,11 @@ func WithRequestSplitPath(splitPath []string) RequestOption {
 // Values set with WithEnv always have priority over automatically populated values.
 func WithRequestEnv(env map[string]string) RequestOption {
 	return func(o *FrankenPHPContext) error {
-		o.env = env
+		e := make(map[string]string, len(env))
+		for k, v := range env {
+			e[k] = v
+		}
+		o.env = e
```

Comment:
This was a nasty bug that I somehow triggered. Essentially, we end up with concurrent map updates when we set `fc.env["FRANKENPHP_WORKER"] = "1"`, which causes a data race. This especially happens in tests because we actually get an environment of `FOO=BAR`, but in real requests, we get an empty environment.

This is more of a "hotfix" to get tests to run/pass on this branch. I'll work on a separate PR to deal with the data race specifically.